### PR TITLE
WIP: Feat Article: "the current state of ai for content creators"

### PR DESCRIPTION
**Pull request by vince-io-onsite**

### CODE REVIEW

**Comment by vince-io-onsite** [comment]
Preview looks different than locally

**Comment by vince-io-onsite** [comment]
@sanderdejong88 @DaveBitter Only in the preview build. Every line inside a <div> is wrapped in a <p>. Any reason for this?

File: data/series/open-ai.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+title: Open AI's _Codex_, _GPT_ and _DALL-E_ explained
+date: '2022-09-12'
+tags: ['development', 'AI']
+images: ['/articles/open-ai/header.png']
+summary: 'In this series we will take a look at three of the most popular AI tools from Open Ai: Codex (aka Github Copilot),
+GPT-3 and DALL-E. We will discuss some of the best use cases for each and how you can get started with them.'
```
Comment (DaveBitter) [inline data/series/open-ai.md]: I'm all for making this a series vs. a Tag, but what's the goal of your series and how do these multiple parts support that? 

Also, could you start/end the articles referring to the previouse/next article?

File: data/blog/open-ai/the-current-state-of-ai-for-content-creators.mdx [hunk 2]
```diff
@@ -0,0 +1,254 @@
+---
+title: 'The current state of AI for __content creators__'
+date: '2022-09-12'
+tags: ['content', 'AI']
+images: ['/articles/open-ai/the-current-state-of-ai-for-content-creators/header.jpeg']
+summary: 'In this blog post, we will take a look at some of the ways that GPT-3 can be used by content creators to
```
Comment (DaveBitter) [inline data/blog/open-ai/the-current-state-of-ai-for-content-creators.mdx]: e.g. "In part two of this series we will take a look at..

File: data/blog/open-ai/the-current-state-of-ai-for-content-creators.mdx [hunk 3]
```diff
@@ -0,0 +1,254 @@
...
+improve their workflow. We will also provide some tips on how to get started with using GPT-3 for your own projects. We
+will use OpenAI’s Playground to try out some of the features of GPT-3.'
+authors: ['vince-liem', 'openai-gpt3']
+theme: 'rouge'
+serie: 'open-ai'
+---
+
+_The author generated this text in part with GPT-3, OpenAI’s large-scale language-generation model. Upon generating
```
Comment (DaveBitter) [inline data/blog/open-ai/the-current-state-of-ai-for-content-creators.mdx]: Could we use one of these to highlight this:

Screenshot 2022-09-20 at 12 33 34

**Pull request merged**